src/ts/storageutils.ts

```diff
@@ -1,36 +1,14 @@
 export namespace StorageUtils {
-  let hasLocalStorageCache: boolean;
-
-  export function hasLocalStorage(): boolean {
-    if (hasLocalStorageCache) {
-      return hasLocalStorageCache;
-    }
-
-    // hasLocalStorage is used to safely ensure we can use localStorage
-    // taken from https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#Feature-detecting_localStorage
-    let storage: any = { length: 0 };
+  export function isLocalStorageAvailable(): boolean {
```

Comment:
Does this need to be exported?

Reply:
Doesn't have to I guess, we only use it inside `storageutils`. Just kept it that way because it was exported before as well:

[Remove export of internal function](https://github.com/bitmovin/bitmovin-player-ui/pull/612/commits/f3941616cacbc95a3e4acb94aa1d62581cd6d428)